### PR TITLE
Fix null username

### PR DESCRIPTION
**Pull request by gguijarro-c-chwy**
Can the username be null? Yes? Then it should not be required in the model.

### CODE REVIEW

**Comment by gguijarro-c-chwy** [comment]
@n13  i suggest you ignore the generated code. I never ever read those files. 
Focus on the code that is used to create the generated code. 

Trying to read or update generated code is a recipe for disaster. Plus not really useful. (In most cases)

**Review by n13 — APPROVED** [review]
LGTM

**Comment by n13** [comment]
> @n13 i suggest you ignore the generated code. I never ever read those files. Focus on the code that is used to create the generated code.
> 
> Trying to read or update generated code is a recipe for disaster. Plus not really useful. (In most cases)

I guess it just made it very difficult to trace the root cause

File: lib/core/network/models/user_profile_data.dart [hunk 1]
```diff
@@ -9,7 +9,7 @@ class UserProfileData with _$UserProfileData {
     required String accountName,
     @Default(null) String? userImage,
     @Default(null) String? bio,
-    required String userName,
```
[the remark this answers is not in the document]
Comment (n13) [reply lib/core/network/models/user_profile_data.dart:12]: Ok

**Comment by n13** [comment]
> @n13 i suggest you ignore the generated code. I never ever read those files. Focus on the code that is used to create the generated code.
> 
> Trying to read or update generated code is a recipe for disaster. Plus not really useful. (In most cases)

Sure - will ignore it.

I still think there's just very small gains from this, and it's making my debugging life harder. 

But let's see.

**Pull request merged**